lua/plugins/languages/typescript.lua

```diff
@@ -30,7 +52,13 @@ return {
       },
     },
   },
-
+  -- {
```

Comment:
actually add this back because this ensures if you don't have vue-language-server installed in mason it'll automatically install it for you. if you already did it manually you wont need to do it again rn but if ur opening vue on another computer u would need to install manually again